src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -410,17 +411,33 @@ private long CreateBackupTask(PeriodicBackup periodicBackup, bool isFullBackup)
 
                 if (_serverStore.Server.CpuCreditsBalance.BackgroundTasksAlertRaised.IsRaised())
                 {
-                    throw new MaxNumberOfConcurrentBackupsException(
+                    throw new BackupDelayException(
                         $"Failed to start Backup Task: '{periodicBackup.Configuration.Name}'. " +
-                        $"The task cannot run because the CPU credits allocated to this machine are nearing exhaustion.");
+                        $"The task cannot run because the CPU credits allocated to this machine are nearing exhaustion.")
+                    {
+                        DelayPeriod = TimeSpan.FromMinutes(10)
```

Comment:
@ayende shouldn't we have this a configurable, different machine might need different TimeSpan to regain CpuCredit.

Reply:
It's a good suggestion, yes

Reply:
Done, please review